src/transformers/cache_utils.py

```diff
@@ -21,6 +21,104 @@
 logger = logging.get_logger(__name__)
 
 
+# Utility functions for static/sliding cache update logic
+def _static_cache_update_logic(
```

Comment:
```suggestion
def _static_cache_update(
```
not sure we need the `logic`